src/main/java/aztech/modern_industrialization/blocks/forgehammer/ForgeHammerScreenHandler.java

```diff
@@ -114,6 +115,14 @@ public boolean mayPlace(ItemStack stack) {
             @Override
             public void onTake(Player player, ItemStack stack) {
                 ForgeHammerScreenHandler.this.onCraft();
+                // Don't play the sound multiple times within the same tick
+                // Prevents the sound being played a lot when shift-clicking the output into your inventory
+                context.execute((world, pos) -> {
+                    if (lastSoundTime < world.getGameTime()) {
+                        world.playSound(null, pos, SoundEvents.SMITHING_TABLE_USE, SoundSource.BLOCKS, 1.0F, 1.0F);
```

Comment:
If this is called both on the client and the server, the `null` should probably be replaced by `player`? Can't check the vanilla code right now... 😉

Reply:
If you were to provide the player it would play the sound twice I believe. Once on the server and on the player, by providing `null` it only plays on the server side. 

On the client side it won't do anything, since player is null it will fail the `if (player == this.minecraft.player)` in `ClientLevel`

Reply:
Also to note, this is the same way the tool break sound effect is played in `onCraft` in this same class [here](https://github.com/AztechMC/Modern-Industrialization/blob/master/src/main/java/aztech/modern_industrialization/blocks/forgehammer/ForgeHammerScreenHandler.java#L228)